### PR TITLE
feat - alert component

### PR DESCRIPTION
**Pull request by prachi-sahani**
Created alerts component
Types of alerts included:
 - outlined alerts based on role/color(primary, danger, warn, success, info)
 - filled alerts 
 - alerts with icon

alert2

Updated:
alert3

### CODE REVIEW

File: css/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@import url(../base/reset.css);
+@import url(../utilities/color.css);
+
+.alert{
+    border: 1px solid;
+    border-radius: 4px;
+    font-size: 1.25rem;
+    padding: 0.5rem;
+    overflow-wrap: anywhere; /* for responsiveness */
```
Comment (ritik-patel05) [inline css/components/alert.css]: overflow-wrap: break-word will be better?
Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

**Review by ritik-patel05 — APPROVED** [review]
Looks good and clean!

**Review by FarhanMobashir — COMMENTED** [review]
make the font size smaller in small screen and also you can lower the padding a little bit

**Review by aryan-13 — COMMENTED** [review]
The code looks good to me!

File: css/components/alert.css (hunk 1 above)
Comment (prachi-sahani) [reply css/components/alert.css]: Done!

**Comment by prachi-sahani** [comment]
> make the font size smaller in small screen and also you can lower the padding a little bit

Done!

**Comment by FarhanMobashir** [comment]
👍👍

**Pull request merged**